nestkernel/connection_manager.cpp

```diff
@@ -1288,33 +1315,35 @@ nest::ConnectionManager::get_sources( std::vector< index > targets,
 void
 nest::ConnectionManager::get_targets( std::vector< index > sources,
   std::vector< std::vector< index > >& targets,
-  index synapse_model )
+  index synapse_model,
+  std::string post_synaptic_element )
 {
-  thread thread_id;
+  unsigned int thread_id;
```

Comment:
you can define this variable in line 1335, and type should be `thread`. I would prefer homogeneous usage of `tid` instead of `thread_id` but maybe this should be part of a maintanance PR, not this one.